ha/setup/create_pacemaker_resources.py

```diff
@@ -44,11 +44,12 @@ def cib_get(cib_xml):
 
 def motr_hax(cib_xml, push=False):
     """Create resources that belong to motr group and clone the group."""
+    cmd_hare_consul = f"pcs -f {cib_xml} resource create hax-consul systemd:hare-consul-agent --group io_group"
```

Comment:
Put a TODO here that it's temporary code and should be removed once fixed by hax/provisioner.

Reply:
Done